config/rbac/rbac_role.yaml

```diff
@@ -88,3 +88,15 @@ rules:
   - update
   - patch
   - delete
+- apiGroups:
```

Comment:
eep, how did this sneak in here? I hope none of the existing provider implementations where relying on this behavior... @vincepri can you think of any reason this would be needed?

Reply:
@detiber thank you, the event rbac rule been moved to example provider config now.